systems/BannerNotifications.css

```diff
@@ -0,0 +1,60 @@
+.wds-banner-notification {
```

Comment:
is this a copy and paste or is this file generated from design-system/style-guide?

Reply:
Files outside of the `source/` directory are built - this is built from the DS while compiling via https://github.com/Wikia/react-design-system/blob/master/source/components/BannerNotification/styles.scss#L1-L7